### ISSUE TITLE
TravisCI is unable to publish new versions

### ISSUE THREAD

**Issue by ps-jay** [reecetech/django_informixdb_vault#8]
```
Installing deploy dependencies
Successfully installed dpl-pypi-1.10.16
1 gem installed
2021-03-25 06:02:12 URL:https://bootstrap.pypa.io/get-pip.py [1927630/1927630] -> "-" [1]
ERROR: This script does not work on Python 2.7 The minimum supported Python version is 3.6. Please use https://bootstrap.pypa.io/pip/2.7/get-pip.py instead.
Couldn't install pip, setuptools, twine or wheel.
```

**Comment by ps-jay** [comment]
Weird ... looking at the TravisCI pipeline, it looks like we're using their standard functionality:
https://github.com/reecetech/django_informixdb_vault/blob/master/.travis.yml#L26-L31
```yaml
deploy:
  provider: pypi
  username: "__token__"
  password: $PYPI_TOKEN
  on:
    branch: master
```

**Comment by ps-jay** [comment]
Resolved in #9
